flink-runtime/src/test/java/org/apache/flink/runtime/highavailability/FileSystemJobResultStoreTest.java

```diff
@@ -44,7 +44,7 @@
 
 /** Tests for the internal {@link FileSystemJobResultStore} mechanisms. */
 @ExtendWith(TestLoggerExtension.class)
-public class FileSystemJobResultStoreTestInternal {
```

Comment:
We might want to come up with a more descriptive name for this test. The test is actually testing the file operations performed by the `FileSystemJobResultStore` implementation. The files function as some kind of user-facing interface as well. That makes it necessary to test this part.

What about renaming it into something like `FileSystemJobResultStoreFileOperationsTest`? WDYT?